test/PerfTests/JavaScript/FanOutFanIn/index.js

```diff
@@ -0,0 +1,11 @@
+﻿const df = require("durable-functions");
+
+module.exports = async function (context, req) {
+    const client = df.getClient(context);
+    const numActivities = req.body;
```

Comment:
Consider validating the input before proceeding - i.e. making sure it's a positive number. It will save folks a bunch of headaches, based on my experience writing similar tests.